### ISSUE TITLE
Support for view completion with compact()

### ISSUE THREAD

**Issue by SudoGetBeer** [laravel-idea/plugin#282]
Hello,

I noticed that auto completion for variables (i.e. model properties) in blade files don't seem to work if the variables are added with the use of compact:
```php
        return view('order.index')
            ->with(compact('orders'));
```

But this does work:

```php
        return view('order.index')
            ->with('orders', $orders);
```

Or is it a wrong configuration on my side?

Best regards

**Comment by adelf** [comment]
The issue here is that I need the type for $orders.

```
@foreach($orders as $order)
  {{ $order-><here it should complete Order methods and fields> }}
```

I can just ask PhpStorm - what is the type of `$orders` variable in the `with('orders', $orders)` call. It's easy. It's harder to find the type for `compact()` call parameters... but I'll try.

**Comment by adelf** [comment]
Implemented in 4.4.4. Thank you. 

https://twitter.com/laravel_idea/status/1462781313581293576
